image_provisioner/playbooks/roles/aos-ansible/tasks/main.yaml

```diff
@@ -1,9 +1,20 @@
 ---
-- name: customize aos-ansible inventory file
-  template: src=hosts.j2 dest={{ aos_ansible_inventory }}
+- block:
+   - name: customize aos-ansible inventory file
```

Comment:
Please use conditionals where necessary in templates rather than having multiple steps and multiple files.